helpers/reference-data-reader/src/index.ts

```diff
@@ -27,3 +37,22 @@ export const getRpcLatestAnswer: ReferenceDataPrice = async (
   const aggregator = AggregatorInterfaceFactory.connect(contractAddress, provider)
   return (await aggregator.latestAnswer()).div(multiply).toNumber()
 }
+
+export const getRpcLatestRound: ReferenceDataRound = async (
+  contractAddress: string,
+  multiply: number,
+): Promise<RoundData> => {
+  const rpcUrl = process.env.RPC_URL
+  const ABI = [
+    'function latestRoundData() external view returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound)',
+  ]
+  const provider = new ethers.providers.JsonRpcProvider(rpcUrl)
+  const contract = new ethers.Contract(contractAddress, ABI, provider)
+  const result = await contract.latestRoundData()
+  result.answer = result.answer / multiply
+  result.startedAt = Number(result.startedAt)
+  result.updatedAt = Number(result.updatedAt)
+  result.answeredInRound = Number(result.answeredInRound)
+  result.roundId = Number(result.roundId)
+  return result
+}
```

Comment:
If you import the V2V3 interface, we can make it a bit better: `import { AggregatorV2V3InterfaceFactory } from '@chainlink/contracts/ethers/v0.6/AggregatorV2V3InterfaceFactory'`

We need to use BigNumber instead of Number as well, as the numbers we're dealing with are too big. This change should return BigNumber. Changes will have to be made in the consumer to reflect this change.

```suggestion
export const getRpcLatestRound: ReferenceDataRound = async (
  contractAddress: string,
): Promise<RoundData> => {
  const rpcUrl = util.getRequiredEnv('RPC_URL')
  const provider = new ethers.providers.JsonRpcProvider(rpcUrl)
  const aggregator = AggregatorV2V3InterfaceFactory.connect(contractAddress, provider)
  return await aggregator.latestRoundData()
}
```

Reply:
changes done